include/core/parameters_lagrangian.h

```diff
@@ -254,7 +254,22 @@ namespace Parameters
       std::string force_torque_output_name;
 
       // Center of mass
-      Point<dim> point_center_mass;
+      Point<dim> triangulation_center_mass;
+
+      // Triangulation inertia
+      Tensor<1, 3> triangulation_inertia;
+
+      // Boundary mass
+      double triangulation_mass;
+
+      // Enable moving boundary due to particles forces collision with walls
+      bool enable_moving_boundary;
```

Comment:
I think it would be better if you chose a better name for this variable like: "free_boundary_motion"

Reply:
Yeah, it would be up to you. Cannot make any change in this branch anymore. But yes, good idea.